src/view/configuration/configuration.jsx

```diff
@@ -108,7 +111,9 @@ const getSettings = ({ values, initInfo }) => {
         "urlDestinationsEnabled",
         "cookieDestinationsEnabled",
         "prehidingStyle",
-        "idMigrationEnabled"
+        "idMigrationEnabled",
+        "clickCollectionEnabled",
+        "downloadLinkQualifier"
```

Comment:
I think we should only add `downloadLinkQualifier` to the settings object if `clickCollectionEnabled` is `true`, since it's useless data otherwise. This is the approach we've taken for `idSyncContainerId` and `context` below.

You'll also need to validate the `downloadLinkQualifier` only when click collection is enabled. You'll see where this is currently a problem if you provide an invalid regular expression pattern, then uncheck the "Enable click data collection" checkbox, then hit save. It doesn't pass validation but there's no indication of what's wrong and, technically, nothing _is_ wrong.